src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random;
+
+    ColorSupplier(Random random) {
+        this.random = random;
+    }
+
     public String getRandomColor() {
-        return null;
+        Color[] values = Color.values();
+        int randomIndex = random.nextInt(values.length);
+        Color randomColor = values[randomIndex];
+        return randomColor.toString();
```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants
```suggestion
        return randomColor.name();
```